fundingmanager.go

```diff
@@ -738,6 +755,17 @@ func (f *fundingManager) Stop() error {
 func (f *fundingManager) stop() error {
 	fndgLog.Infof("Funding manager shutting down")
 
+	// Locate the RPCAcceptor within the ChainedAcceptor and stop it if it exists.
+	openChanPredicate := f.cfg.OpenChannelPredicate
+	chainedAcceptor, _ := openChanPredicate.(*chanacceptor.ChainedAcceptor)
+	for _, acceptor := range chainedAcceptor.Acceptors {
+		rpcAcceptor, ok := acceptor.(*chanacceptor.RPCAcceptor)
+		if ok {
+			rpcAcceptor.Stop()
```

Comment:
Abstraction leak here, also same comment about it already being stopped (or stopped elsewhere) at this point.